src/pages/library/DataBrowser.js

```diff
@@ -55,12 +57,12 @@ const extractCatalogFilters = dataCatalog => {
     labels: getUnique(dataset => getConsortiumsFromDataset(dataset), dataCatalog)
   }, {
     name: 'Data use policy',
-    labels: getUnique('dataReleasePolicy.policy', dataCatalog),
+    labels: getUnique(dataset => getDataReleasePolicyFromDataset(dataset).policy, dataCatalog),
     labelRenderer: rawPolicy => {
-      const { label, desc } = datasetReleasePolicies[rawPolicy] || datasetReleasePolicies.releasepolicy_other
+      const { label, desc } = datasetReleasePolicies[rawPolicy] || datasetReleasePolicies.unknownReleasePolicy
```

Comment:
Why is `|| unknownReleasePolicy` needed here? It looks like `getDataReleasePolicyFromDataset()` will already return this value in the unknown case.

Reply:
Going to try to address this with the bigger overhaul. Let me do that and then I will ping for another round of review